### DISCUSSION TITLE
Add versioning to built files

### DISCUSSION THREAD

**Discussion by fartymonk** [vercel/next.js#22637] [Help]
Hi, I faced a problem with nextjs site. During the course of development, there are times when the site are deployed for review. However it seemed that when huge update are applied to the existing deployed site, users accessing the site will face many errors due to their browser cache having the old built js. I can't expect users to clear their browser cache whenever updates are applied, and clearing cache from ie. Cloudflare or server does not help.

I know I could just suffix-ed the files with version or whatever just so the browser sees it as a new file and retrieve it from server instead of local cache. But, how do I do that in NextJS?

**Comment by antonioberetini** [comment]
You might find this helpful: https://dev.to/ammartinwala52/clear-cache-on-build-for-react-apps-1k8j
